# This patch is based on SphinxSearch REL1_27.tar.gz
# We remove a regex caret to enable substring searches of the title.
--- SphinxMWSearch.php.orig     2025-04-07 10:26:47.892625022 -0700
+++ SphinxMWSearch.php  2025-04-07 10:27:00.853910935 -0700
@@ -45,7 +45,7 @@
                $search_engine = new SphinxMWSearch( wfGetDB( DB_SLAVE ) );
                $search_engine->namespaces = $namespaces;
                $search_engine->setLimitOffset( $limit, $offset );
-               $result_set = $search_engine->searchText( '@page_title: ^' . $term . '*' );
+               $result_set = $search_engine->searchText( '@page_title: ' . $term . '*' );
                $results = array();
                if ( $result_set ) {
                        while ( $res = $result_set->next() ) {